app/models/event_stream.rb

```diff
@@ -130,7 +78,6 @@ def group_level
     @group_and_level ||= self.class.group_and_level(event_type)
```

Comment:
Should there be an abstract version of group_and_level at the class level in the EventStream base class?

Reply:
👇